lesson2 HW

atni1 opened the pull request:
uncomment in turn

atni1 committed:
lesson2 HW

uncomment in turn

Om1X commented on Lesson2/script.js:
```diff
@@ -0,0 +1,111 @@
+} else if (a < 0 && b < 0){
+        x = a * b
+        alert(x);
+} else if (a >= 0 && b < 0 || a <= 0 && b > 0) {
```
if тут можно опустить

Om1X commented on Lesson2/script.js:
```diff
@@ -0,0 +1,111 @@
+// function mathOperation(arg1, arg2, operation) {
+//     switch (operation) {
+//         case 'sum':
+//             return arg1 + arg2;
+//             break;
+//         case 'diff':
+//             return arg1 - arg2;
+//             break;  
+//         case 'split':
+//             return arg1 / arg2;
+//             break;  
+//         case 'multiplication':
+//             return arg1 * arg2;
+//             break;         
+//         default: 
+//             break;
```
не использовали функции из предыдущего задания

atni1 committed:
changes

Не понял до конца что делать с if  в задании №3, как написать без него. Задание №6 исправлено
